README.md

```diff
@@ -27,6 +27,11 @@ If you want to run it locally
   - The page uses Jekyll's `site.github.public_repositories` object to get the repo data. This data is only available when live.
   - Custom repo data is generated by Github's JS API, so it will show up locally.
 
+### Content Management
+- The content for each project comes from the GitHub Pages for that repo.
+This can be from a `gh-pages` branch, or from the `/docs` directory, based on the GitHub Pages settings.
```

Comment:
Maybe link this to the GitHub documentation on the topic?

https://help.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site

Reply:
👍 Fixed in 2a49d198702382495362a7e0cbab17857c453ea4